games/o.yaml

```diff
@@ -145,6 +145,16 @@
     'https://a.fsdn.com/con/app/proj/o2r/screenshots/255628.jpg']
   remakes: [Rodent's Revenge]
 
+- name: OpenSC2K
+  development: active
+  lang: JavaScript
+  url: https://github.com/rage8885/OpenSC2K
+  repo: https://github.com/rage8885/OpenSC2K
+  updated: 2018-02-09
+  images: ['https://github.com/rage8885/OpenSC2K/blob/master/screenshots/1.png', 'https://github.com/rage8885/OpenSC2K/blob/master/screenshots/2.png',
```

Comment:
I think these image links should point to the raw images https://stackoverflow.com/a/39066549/2038264